main/lsp/LSPPreprocessor.cc

```diff
@@ -266,16 +177,7 @@ void LSPPreprocessor::preprocessAndEnqueue(QueueState &state, unique_ptr<LSPMess
             break;
         }
         case LSPMethod::Initialized: {
-            InitializedParams &params = *get<unique_ptr<InitializedParams>>(msg->asNotification().params);
```

Comment:
We now perform indexing and file hashing via the `IndexAndHashWorkspaceTask` run on the typechecker thread. This is the blocking part of `initialization` that we talked about before.